app/common/elasticsearchclient.py

```diff
@@ -604,30 +604,19 @@ def get_gene_info(self, gene_ids, **kwargs):
             params.datastructure = SourceDataStructureOptions.FULL
         source_filter = SourceDataStructureOptions.getSource(params.datastructure)
         if params.fields:
-            source_filter["include"] = params.fields
+            source_filter["includes"] = params.fields
```

Comment:
include**s** implies plural. would change it back to ""include""

Reply:
@elipapa that is the point. they changed from singular to plural. why? are they bored?. So the right word is "includes" there is no other word for that.